docs/snapcraft-advanced-features.md

```diff
@@ -11,16 +11,16 @@ Our showcase can be found in the actual source of `snapcraft` itself. Check
 it out by simply running:
 
 	git clone https://github.com/ubuntu-core/snapcraft
-	cd snapcraft/examples
+	cd snapcraft/demos
```

Comment:
This is weird given that we have `debian/snapcraft-examples.examples` which has `demos/*` in it which makes it so that the files get packaged in `/usr/share/doc/snapcraft-examples/examples/...`

We would need to get rid of this and use a regular `debian/.*.install` rule for this.